src/components/PoolOverview.scss

```diff
@@ -56,32 +47,41 @@ div.poolOverview {
 
     & > img {
       margin-right: 4px;
-      vertical-align: text-top;
+      vertical-align: text-bottom;
     }
   }
 
   div.right {
-    & > div {
-      margin-right: 40px;
+    display: grid;
```

Comment:
I think we also want a `justify-content: flex-end;` here as well to pull the content right

Reply:
Just did it. Forgot about the right align part.